cmd/root_runner.go

```diff
@@ -66,8 +68,18 @@ func runRoot(cmd *cobra.Command, args []string) error {
 		textErr := text.CheckMessageTitle(commitObject.Message)
 
 		if textErr != nil {
-			return textErr
+			faultyCommits = append(faultyCommits, commitHash)
+		}
+	}
+
+	if len(faultyCommits) != 0 {
+		for _, commitHash := range faultyCommits {
+			log.Printf("Commit %v is not conventional commit compliant", commitHash)
 		}
+
+		log.Printf("%v of %v commits are conventional commit compliant", len(commits)-len(faultyCommits), len(commits))
+
+		return nil
```

Comment:
This line should return an error. This way the command is killed and CI will fail.

Reply:
@fallion 🚀